services/api/src/rsu_bsmdata.py

```diff
@@ -2,96 +2,184 @@
-    result.append({
-        "type": "Feature",
-        "geometry": {
-            "type": "Point",
-            "coordinates": [
-                row["long"],
-                row["lat"]
-            ]
-        },
-        "properties": {
-            "id": row["Ip"],
-            "time": util.format_date_denver_iso(row["time"])
-        }
-    })
-    count += 1
-
-  logging.info(f"Query successful. Record returned: {count}")
-
-  return result, 200
 
 # REST endpoint resource class and schema
 from flask import request
 from flask_restful import Resource
 from marshmallow import Schema, fields
 
+
 class RsuBsmDataSchema(Schema):
-  geometry = fields.String(required=False)
-  start = fields.DateTime(required=False)
-  end = fields.DateTime(required=False)
+    geometry = fields.String(required=False)
+    start = fields.DateTime(required=False)
+    end = fields.DateTime(required=False)
+
 
 class RsuBsmData(Resource):
-  options_headers = {
-    'Access-Control-Allow-Origin': os.environ["CORS_DOMAIN"],
-    'Access-Control-Allow-Headers': 'Content-Type,Authorization',
-    'Access-Control-Allow-Methods': 'POST',
-    'Access-Control-Max-Age': '3600'
-  }
-
-  headers = {
-    'Access-Control-Allow-Origin': os.environ["CORS_DOMAIN"],
-    'Content-Type': 'application/json'
-  }
-
-  def options(self):
-    # CORS support
-    return ('', 204, self.options_headers)
-
-  def post(self):
-    logging.debug("RsuBsmData POST requested")
-    
-    # Get arguments from request
-    try:
-      data = request.json
-      pointList = data['geometry']
-      start = data['start']
-      end = data['end']
-    except:
-      return ('Body format: {"start": string, "end": string, "geometry": coordinate list}', 400, self.headers)
-    
-    data, code = query_bsm_data(pointList, start, end)
-    return (data, code, self.headers)
\ No newline at end of file
+    options_headers = {
+        'Access-Control-Allow-Origin': os.environ["CORS_DOMAIN"],
+        'Access-Control-Allow-Headers': 'Content-Type,Authorization',
+        'Access-Control-Allow-Methods': 'POST',
+        'Access-Control-Max-Age': '3600'
+    }
+
+    headers = {
+        'Access-Control-Allow-Origin': os.environ["CORS_DOMAIN"],
+        'Content-Type': 'application/json'
+    }
+
+    def options(self):
+        # CORS support
+        return ("", 204, self.options_headers)
+
+    def post(self):
+        logging.debug("RsuBsmData POST requested")
+
+        # Get arguments from request
+        try:
+            data = request.json
+            pointList = data["geometry"]
+            start = data["start"]
+            end = data["end"]
+        except:
+            return (
+                'Body format: {"start": string, "end": string, "geometry": coordinate list}',
+                400,
+                self.headers,
+            )
+        db_type = os.getenv("COUNTS_DB_TYPE", "BIGQUERY").upper()
+        data = []
+        code = None
+
+        if db_type == "BIGQUERY":
```

Comment:
Might be good to prevent configuration errors here if we swapped the db_type check around and checked for MONGODB first, then just fell into an "else" statement instead of elif. It's possible here if someone fat-fingered the counts_db_type in their config that it would error out and we wouldn't have a default to fall back on in this case.

Reply:
Addressed